client/lib/degrade-es6-fns.js

```diff
@@ -0,0 +1,20 @@
+import partial from 'lodash/partial';
+
+function wrapFnWithWarning( fn, name ) {
+	const consoleFn = ( console.error || console.log ).bind( console );
+	return function() {
+		const err = new Error( `${ name } is not supported on all browsers. You must use a replacement method from lodash.` );
+		consoleFn( err );
+		fn.apply( this, arguments );
```

Comment:
We should return the value of the original function. For example, `[ 'foo' ].keys()` should return an array iterator.